cvat/apps/engine/views.py

```diff
@@ -2282,7 +2282,14 @@ def perform_update(self, serializer):
                 "Sublabels cannot be modified this way. "
                 "Please send a PATCH request with updated parent label data instead.",
                 code=status.HTTP_400_BAD_REQUEST)
-
+        attributes = self.request.data.get('attributes')
```

Comment:
We need not only run this validation when updating a Label, but also when creating it. 

Please, move this logic to the `LabelSerializer` (there are methods `update_label` and `create_labels`).
These methods are called from ProjectWriteSerializer and TaskWriteSerializer when labels updated.